recipes/uncrustify/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "uncrustify" %}
+{% set version = "0.69.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/uncrustify/uncrustify/archive/{{ name }}-{{ version }}.tar.gz
+    sha256: 33bd97a07f7c4bd114874f73171aca220bf05c17108f8505a117b97374a347b6
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
+    - vs2017_win-32                      # [win32]
```

Comment:
Someone from @conda-forge/staged-recipes with knowledge of windows will ned to verify these lines.

Reply:
If we don't get a comment by the end of today, we can merge.